src/main/constants.ts

```diff
@@ -0,0 +1 @@
+export const DOWNLOAD_URL = "https://download.nine-chronicles.com";
```

Comment:
It may be better to put this alongside with updater source code, although I'm not so sure about this suggestion. Other constants are usually stored inside `config.ts`, but that doesn't look like a good place either.

Reply:
I had hardcoded it because... I didn't have a good place to put it, but it's not all that bad to add `DownloadBaseURI` to `config.ts`

Reply:
Yeah if possible I think it makes sense to put in there.